app/customer_routes.py

```diff
@@ -0,0 +1,152 @@
+from app import db
+from app.models.customer import Customer
+from flask import Blueprint, json, jsonify, request
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime
+import requests
+import os
+
+from tests.test_wave_01 import CUSTOMER_NAME
+
+customer_bp = Blueprint("customers", __name__, url_prefix=("/customers"))
+
+
+@customer_bp.route("", methods=["GET"])
+def get_customer():
+    if request.method == "GET":
+        customers = Customer.query.all()
+        customer_response = []
+        for customer in customers:
+            customer_response.append(
+                {
+                    "id": customer.id,
+                    "name": customer.name,
+                    "postal_code": customer.postal_code,
+                    "phone": customer.phone,
+                }
+            )
+    return jsonify(customer_response), 200
+
+
+@customer_bp.route("", methods=["POST"])
+def put_customer():
```

Comment:
i would rename this to something like `post_customer()` because this a post method